design-proposals/vm-console-logs.md

```diff
@@ -0,0 +1,743 @@
+# Overview
+Linux and Windows Server VMs can log boot and application data to their serial console.
+KubeVirt user can great benefit from having VMs logs, collected over the serial console, integrated into the Kubernetes logging architecture. 
+
+## Motivation
```

Comment:
- Are there similar solutions for other VM management systems?
- Were there specific requests for something like this from users?

Reply:
> * Are there similar solutions for other VM management systems?

This is a pretty common pattern across all the most popular HyperScalers, see as examples:
- https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-console.html#instance-console-console-output
- https://learn.microsoft.com/en-us/azure/virtual-machines/boot-diagnostics
- https://cloud.google.com/compute/docs/troubleshooting/viewing-serial-port-output#enable-stackdriver
...

> * Were there specific requests for something like this from users?

yes, https://groups.google.com/g/kubevirt-dev/c/qBdboekQ_Vk for instance

Reply:
Great, thank you!
It will be valuable to add this to the design. I guess it is a sub-section of the motivation.

Reply:
done